### ISSUE TITLE
Mobile App Inbox: Unable to assign viewDelegate from Objective C

### ISSUE THREAD

**Issue by AKhannaEpay** [Iterable/swift-sdk#567]
Hi Team,

We are using Objective c for our project and want to use the Mobile App Inbox in our application but we are facing below challenge with the SDK.

1) There is no API documentation on API Reference section of Mobile App Inbox website where to set the delegate in objective C

2) Can't see any other way to use or implement the Delegate for Objective C which is used for setting the viewDelegate except class name.

3) It would be nice if there is a link or something that can help us on how to implement the changes i requested above with Objective C or by making minor change from your side depicted below.

**Current implementation**

```
public var viewDelegate: IterableInboxViewControllerViewDelegate? {
        didSet {
            inboxViewController?.viewDelegate = viewDelegate
        }
}
```

**Requested change**

```
@objc public var viewDelegate: IterableInboxViewControllerViewDelegate? {
        didSet {
            inboxViewController?.viewDelegate = viewDelegate
        }
    }
```

I request you to please consider this request on priority and keep me posted on this.

**Comment by tapashmajumder** [comment]
@AKhannaEpay Thanks for raising this issue. It appears that we missed this. I hope you have taken a look at sample-apps/inbox-customization project to see the recommended way to do inbox customization. 

@roninopf Can you please take a look at this?

**Comment by AKhannaEpay** [comment]
@tapashmajumder: Yes i looked the inbox customisation very well but stuck on this point as it is not supporting in Objective C. It will be very helpful if you can expedite this change for release. 

Thanks

**Comment by roninopf** [comment]
@AKhannaEpay We released 6.4.6 yesterday with this fix, can you confirm if it works for your use case? Thanks for reporting!

**Comment by AKhannaEpay** [comment]
Hi @roninopf : Its working as expected. Thank you so much for expediting this release.
